docs/source/tracking.rst

```diff
@@ -433,3 +433,39 @@ then make API requests to your remote tracking server.
     with mlflow.start_run():
         mlflow.log_param("a", 1)
         mlflow.log_metric("b", 2)
+
+.. _system_tags:
+
+System tags
+===========
+Runs can be annotated with arbitrary tags. Tag keys that start with ``mlflow.`` are reserved for
+internal use. The following tags are set automatically by MLflow, when appropriate:
+
++-------------------------------+----------------------------------------------------------------------------------------+
+| Key                           | Description                                                                            |
++===============================+========================================================================================+
+| ``mlflow.runName``            | Human readable name that identifies this run.                                          |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.parentRunId``        | The ID of the parent run, if this is a nested run.                                     |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.user``               | Identifer of user who created the run.                                                 |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.type``        | Source type (possible values are ``"NOTEBOOK"``, ``"JOB"``, ``"PROJECT"``,             |
+|                               | ``"LOCAL"``, and ``"UNKNOWN"``)                                                        |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.name``        | Source identifier (e.g., GitHub URL, local Python filename, name of notebook           |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.git.commit``  | Commit hash of the executed code, if in a git repository.                              |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.git.branch``  | Name of the branch of the executed code, if in a git repository.                       |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.source.git.repoURL`` | URL that the executed code was cloned from.                                            |
++-------------------------------+----------------------------------------------------------------------------------------+
+| ``mlflow.project.env``        | One of "docker" or "conda", indicating the runtime context used by the mlflow project. |
```

Comment:
As per offline conversation: Reminder to link Projects page for more details.
also for `entryPoint`